framework/dora_api/infrastructure/middleware.py

```diff
@@ -38,18 +39,42 @@ async def deserialise_web_request():
     _RequestEndpoint = request.endpoint.split(".")[-1]
     if _RequestEndpoint in REQUEST_BODYS_BY_ENDPOINT:
         _RequestData: dict = request.get_json()
+        _DeserialisedRequestData : dict = dict()
 
         for _AttributeName, _AttributeType in get_type_hints(REQUEST_BODYS_BY_ENDPOINT[_RequestEndpoint]).items():
-            _Data = _RequestData[_AttributeName]
+            _Data = _RequestData.get(_AttributeName)
+
+            _AttributeTypeOrigin = get_origin(_AttributeType)
+            if _AttributeTypeOrigin is not None and issubclass(_AttributeTypeOrigin, AttributeChangeTracker):
+                __deserialise_attribute_change_tracker(_AttributeName, _RequestData, _DeserialisedRequestData)
+                continue
```

Comment:
I think the continue makes it confusing, probably better to split the two paths with an if else

Reply:
done